extensions/panache/panache-common/deployment/src/main/java/io/quarkus/panache/common/deployment/PanacheEntityEnhancer.java

```diff
@@ -189,13 +190,15 @@ private void generateMethod(MethodInfo method, AnnotationValue targetReturnTypeE
             mv.visitEnd();
         }
 
-        protected abstract String getModelDescriptor();
+        protected String getModelDescriptor() {
```

Comment:
This is not a correct default for all implementations.

Reply:
when I looked at the 3ish current implementations they all returned the same value.  I can change it back but it just seemed redundant.

Reply:
The `panache-rx` impl (in a branch ATM) doesn't, that's why.

Reply:
ah.  does it *hurt* having that as the default that can be optionally overridden, though?  Seems like pointless work to push downstream if most are always going to be the same.  I'm happy to revert that if you'd like but it seems cleaner this way.

Reply:
No, I suppose it won't hurt. Fine.